includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -277,6 +272,39 @@ public function get_product_data() {
 		return apply_filters( 'wc_stripe_payment_request_product_data', $data, $product );
 	}
 
+	/**
+	 * JS params data used by cart and checkout pages.
+	 *
+	 * @param array $data
+	 */
+	public function get_checkout_data() {
```

Comment:
I think it would be nice to have a simple unit test for this method.

Reply:
Unit tests in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3560/commits/c9c7efe72b7a5e403d806ad935bcee8392d9b5a0, thanks!